polyfills/String/prototype/padEnd/config.json

```diff
@@ -0,0 +1,21 @@
+{
+	"aliases": [
+		"es7"
+	],
+	"browsers": {
+		"android": "*",
+		"bb": "*",
+		"chrome": "<57",
+		"firefox": "<48",
+		"opera": "<44",
+		"safari": "<10",
+		"ie": "<15",
```

Comment:
@JakeChampion I'm assuming this just includes Edge in with IE rather that having them as "separate" browsers?